traveller/modules/cfp/view.py

```diff
@@ -55,6 +57,12 @@ def add_talk(year):
     talk.year = year
     talk.talk_conference = conf
 
+    start_time, end_time = request.form.get('start_time', "09:00"), request.form.get('end_time', "18:00")
```

Comment:
just a confirmation, if someone wants to add a time before 09:00 can it be done? the 09:00 is just a default value?

Reply:
The 09:00 is a just default value but you can't add a time before 9:00. This setting can be changed, though, depending on the time intended time you guys want the conference to start.

Reply:
@iSOLveIT keep it flexible so that we can add times whenever we wish.

Reply:
Can you explain further? I don't really get what you mean by that statement.  
I wanted to make it flexible, but there is no field in the database which states the start and end times for a conference. Would have used that.